### ISSUE TITLE
Privacy questions (add to FAQ?)

### ISSUE THREAD

**Issue by karelbilek** [m2049r/xmrwallet#194]
I would like to know what privacy drawbacks does using this wallet have, if any. Compared to running a full node.

Am I sending my viewkey to the server? Can the server know, which transactions belong to me? If the server is controlled by an adversary, what can he do/know.

I cannot find any authoritative answers.

**Comment by SamsungGalaxyPlayer** [comment]
@karel-3d the privacy implications are the same as using a remote node. [Stack Exchange](https://monero.stackexchange.com/questions/38/what-privacy-or-security-trade-offs-are-associated-with-not-running-your-own-ful/86#86) is a better spot for these types of questions, including questions regarding Monerujo.

I assume the discussion here can continue regarding whether adding this info to the FAQ is necessary. I suggest including a warning suggesting people to use a remote node they control.

**Comment by m2049r** [comment]
Yes, this should definitely be in the FAQ - as soon as I find an answer! All answers to this are sort-of true but far from authoritative or complete IMO.
I feel we should have a complete & true answer. Help!

**Comment by eloyesp** [comment]
@m2049r It could help also to have some instructions on how to setup the server (and the recommended options)

**Comment by m2049r** [comment]
@eloyesp i have written up a pretty detailed account of how to do this here: https://www.reddit.com/r/Monerujo/comments/9obruz/running_a_monero_node/ - not sure it should be part of the repo here - i will make it into a medium article when i get around to it.

**Comment by eloyesp** [comment]
@m2049r it definitely should be part of the repo, or at least linked on the README.
